docs/dev/developer-guides/building-on-zksync/tools.md

```diff
@@ -0,0 +1,17 @@
+# Choosing the right tool
```

Comment:
Honestly... I think we should remove this whole page... it is not adding any value